MilitaryManager.Core/Exceptions/NotFoundException.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+
+namespace MilitaryManager.Core.Exceptions
+{
+    public class NotFoundException : Exception
```

Comment:
Do we need this exception? Is it handled by ExceptionHandlerMiddleware?

Reply:
Now it handles